lib/cli.js

```diff
@@ -9,8 +9,19 @@ const configuration = getConfig();
 const choicesList = choices(configuration);
 const questionsList = questions(choicesList);
 
-inquirer.prompt(questionsList).then((answers) => {
-  const message = answers.moreInfo ? `${answers.editor}` : `${answers.type} ${answers.description}`;
+const prompt = inquirer.prompt(questionsList);
+
+prompt.ui.process.subscribe((answer) => {
+  if (answer.name === 'commitMessageWarning') {
+    const continueExecution = answer.answer;
+    if (!continueExecution) {
+      process.exit();
```

Comment:
- My reasoning for using no exit code was that it didn't end unexpectedly because an error occurred, but rather because the user made the decision to stop it.
- If we use validate to check the length, it's not a warning anymore, but an enforced rule.